### ISSUE TITLE
Document supported platforms

### ISSUE THREAD

**Issue by springmeyer** [mapbox/tippecanoe#508]
We should add to the readme the platforms that are supported. My sense is they are:

 - ubuntu >= 12.04 (with c++11 capable compiler and standard library)
 - centos7
 - >= OS X 10.10

@ericfischer - can you confirm this listing and build some mention into the docs?

**Comment by ericfischer** [comment]
Good point. I'm not even really sure what the minimum Ubuntu and MacOS versions that will work are, since my own testing is all on Ubuntu 14.04 and MacOS 10.11.

**Comment by springmeyer** [comment]
Good point @ericfischer . How about we call "supported platforms" == "ones we test on continuously", so if we don't test it on travis we don't list it? If we want to support OS X 10.10 we can use `osx_image: xcode6.4` per https://docs.travis-ci.com/user/languages/objective-c/#Supported-Xcode-versions.

**Comment by ericfischer** [comment]
That seems appropriate.

Reading the travis documentation I still don't see where you specify which Ubuntu version you want to run under. Do you know where that part is?

**Comment by springmeyer** [comment]
> Reading the travis documentation I still don't see where you specify which Ubuntu version you want to run under. Do you know where that part is?

You supply either `dist:trusty` or `dist:precise`. The former is the default since https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming
